pkg/components/storage.go

```diff
@@ -72,7 +72,7 @@ func startCSIPlugin(cfg *config.MicroshiftConfig, kubeconfigPath string) error {
 	)
 
 	if err := lvmd.LvmSupported(); err != nil {
-		klog.Warning("skipping CSI deployment: %v", err)
+		klog.Warningf("skipping CSI deployment: %v", err)
```

Comment:
I'm not 100% on how klog handles error types, but I think we should change the `%v` to `%w`. The go team has a nice little FAQ on some of these conventions. [see](https://github.com/golang/go/wiki/ErrorValueFAQ#i-am-already-using-fmterrorf-with-v-or-s-to-provide-context-for-an-error-when-should-i-switch-to-w)